File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseRoute.scala

package org.openapitools.client.model

import java.time.OffsetDateTime

case class ResponseRoute (
    _departureTime: OffsetDateTime,
    _arrivalTime: OffsetDateTime,
    _parts: List[ResponseRoutePart]
)
object ResponseRoute {
    def toStringBody(var_departureTime: Object, var_arrivalTime: Object, var_parts: Object) =
        s"""
        | {
        | "departureTime":$var_departureTime,"arrivalTime":$var_arrivalTime,"parts":$var_parts
        | }
        """.stripMargin
}

File: fsharp-giraffe-server/traveltimeplatform/src/model/ResponseRoute.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.ResponseRoutePart

module ResponseRoute = 

  //#region ResponseRoute


  type ResponseRoute = {
    DepartureTime : DateTime;
    ArrivalTime : DateTime;
    Parts : ResponseRoutePart[];
  }
  //#endregion
  